specs/Daito/Log_specification.md

```diff
@@ -0,0 +1,52 @@
+# Log specification
+**Authors:** [liuqinguestc](https://github.com/liuqinguestc)
+
+## Summary
+
+OpenSDS log is not uniform and standardized, and the log format is various.We need to develop a unified specification log format to facilitate location and troubleshooting.
```

Comment:
Add a space after period.

"facilitate location" does not seem right.  Do you mean "store logs at a common location"?

Reply:
multi-cloud project logs can be stored at /var/log/multi-cloud, opensds project logs can be stored aat /var/log/opensds,  We can talk about it.